.github/ISSUE_TEMPLATE/Platform _Epic_Template.md

```diff
@@ -0,0 +1,24 @@
+---
+Name: Epic template
+About: Template for creating  platform epics
+Title: "[EPIC]"
+Labels: epic platform
+
+---
+
+## Outcome/Objective ##
+
+## Description ##
+
+## Product Requirement(s) ##
+
+## Use Case ##
+
+## Dependencies ##
+
+## Acceptance criteria  ##
+
+## Technical Requirement(s)##
+
+## Technical Constraint(s)##
```

Comment:
Not sure what would go here? Maybe remove and defer to the Technical Requirement(s) above.